dependabot/update_lag_status.py

```diff
@@ -0,0 +1,290 @@
+import github
+from github import Github, InputGitAuthor, GithubException
+import json
+import sys
+import os
+from datetime import datetime
+import urllib.request
+import base64
+import matplotlib.image as mpimg
+
+packageUser = os.environ["packageUser"]
+packagePAT = os.environ["packagePAT"]
+packageEmail = os.environ["packageEmail"]
+
+ENCODING = "utf-8"
+ORGANIZATION = "ballerina-platform"
+EXTENSIONS_LIST_FILE = "dependabot/resources/extensions.json"
+BALLERINA_LANG_VERSION_FILE = "dependabot/resources/latest_ballerina_lang_version.json"
+PROPERTIES_FILE = "gradle.properties"
+README_FILE = "README.md"
+LANG_VERSION_KEY = "ballerinaLangVersion"
+github = Github(packagePAT)
+
+all_modules = []
+
+MODULE_NAME = "name"
+ballerina_timestamp = ""
+ballerina_lang_version = ""
+
+def main():
+    readMe_repo = github.get_repo(ORGANIZATION + "/ballerina-release")
+
+    readme_file = get_readme_file()
+    updated_readme = readme_file
+
+    get_lang_version()
+    module_list = get_module_list()
+
+    updated_readme = return_updated_readme(readme_file)
+
+    import matplotlib.pyplot as plt
```

Comment:
Move imports to the top of the file as per standard

Reply:
8bc3836c253f9584162fce9532c843628a4c9cf6

I avoided showing the graph for the moment since I could not resolve the image getting broken issue.